pymc/model_graph.py

```diff
@@ -125,14 +125,17 @@ def make_compute_graph(
 
         return input_map
 
-    def _make_node(self, var_name, graph, *, formatting: str = "plain"):
-        """Attaches the given variable to a graphviz Digraph"""
+    def _make_node(self, var_name, graph, *, nx=False, cluster=False, formatting: str = "plain"):
```

Comment:
it might be pleasant to update the signature to just take `graph_node_fn`, which is either `graph.add_node` or `graph.node`, since they both appear to do this update in place. then we do not need a specific `nx` argument here.

Reply:
Perhaps that is a future improvement?

Reply:
sounds good to me!